tools/mason/MasonHelp.chpl

```diff
@@ -132,6 +134,24 @@ proc masonSearchHelp() {
   writeln(desc);
 }
 
+proc masonModifyHelp() {
+  writeln("Modify a Mason package's Mason.toml");
+  writeln();
+  writeln("Usage:");
+  writeln("    mason rm [options] package");
+  writeln("    mason add [options] package@version");
+  writeln();
+  writeln("Options:");
+  writeln("    -h, --help                  Display this message");
+  writeln("        --external              Add/Remove dependency from external dependencies");
+  writeln("        --system                Add/Remove dependency from system dependencies");  
+  writeln();
+  writeln("Not listing an option will add/remove the dependency from the Mason [dependencies] table");
+  writeln("Versions are not necessary for removing");
```

Comment:
Design: Can `mason add <package>` without a version number just default to the latest version?

Reply:
There's an open design question about whether or not `mason add/rm` should validate packages or not. After some discussion, we decided to not validate packages, therefore `mason add/rm` will not know what the latest version number.

I'd suggest the following edit to the help output:

> Versions are not necessary for removing

```
Versions are necessary for adding dependencies, but not for removing dependencies
Package names and versions are not validated upon adding
```